.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Set up JDK 21
```

Comment:
Nitpick, but I'd name this step correctly for Java 11.

Reply:
Done in 87b40925c8fd1205ad8a312cd1bece85ab12d1d2